sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -114,7 +114,7 @@ abstract class WindowExecBase(
 
   /**
    * Collection containing an entry for each window frame to process. Each entry contains a frame's
-   * [[WindowExpression]]s and factory function for the WindowFrameFunction.
+   * [[WindowExpression]]s and factory function for the [[WindowFrameFunction]].
```

Comment:
This is normally fine, but I know we have run into problems with some scaladoc links to classes. Are you sure this generates OK? probably so, just always get nervous about it.

Reply:
Aside from linking issues, I think it will look more consistent.